core/src/test/java/com/graphhopper/reader/dem/BridgeElevationInterpolatorTest.java

```diff
@@ -75,17 +73,19 @@ public void interpolatesElevationOfPillarNodes() {
         na.setNode(8, 30, 10, 10);
         na.setNode(9, 40, 10, 0);
 
-        EdgeIteratorState edge01 = graph.edge(0, 1, 10, true);
-        EdgeIteratorState edge12 = graph.edge(1, 2, 10, true);
-        EdgeIteratorState edge23 = graph.edge(2, 3, 10, true);
-        EdgeIteratorState edge34 = graph.edge(3, 4, 10, true);
-        EdgeIteratorState edge56 = graph.edge(5, 6, 10, true);
-        EdgeIteratorState edge67 = graph.edge(6, 7, 10, true);
-        EdgeIteratorState edge78 = graph.edge(7, 8, 10, true);
-        EdgeIteratorState edge89 = graph.edge(8, 9, 10, true);
-        EdgeIteratorState edge17 = graph.edge(1, 7, 10, true);
-        EdgeIteratorState edge27 = graph.edge(2, 7, 10, true);
-        EdgeIteratorState edge37 = graph.edge(3, 7, 10, true);
+        FlagEncoder encoder = encodingManager.getEncoder("car");
+        EdgeIteratorState edge01 = GHUtility.setProperties(graph.edge(0, 1).setDistance(10), encoder, 60, true, true);
```

Comment:
What I like here is that the speed is now explicit, while previously one always had to keep in mind that its 60 in these tests. Maybe I just have to get used to this verbose code to create a testing graph, but also see my other comment about `setProperties` which I do not find intuitive.

Reply:
I also do not find it intuitive and for tests we should think about a better solution. Maybe we can use the method you proposed above but with a GraphHopperStorage which gives us access to the EncodingManger where we then by default fetch car?